ClientUI.Components/Portfolio/PortfolioRes.fr.resx

```diff
@@ -0,0 +1,138 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
We may want to remove this file or change it to a JSON file since we are using JSON-based localization.